File: C++_Programming/Practice/Program1.cpp
/*
  Write A program to Add two numbers 
*/

#include<iostream>
using namespace std;

int Addition(int iNo1, int iNo2)
{
  int iAns = 0;

  iAns = iNo1 + iNo2;

  return iAns;
}

int main()
{
  int iValue1 = 20;
  int iValue2 = 4;
  int iRet = 0;

  iRet = Addition(iValue1, iValue2);

  cout<<"Addition is : %d\n"<<iRet;
  
  return 0;
}

File: Java_Programming/Practice/Program1.java
/*
  Program For Subtracting Two numbers  
*/
class Subtraction
{  
  public int Display(int iNo1, int iNo2)
  {
    int Ans = 0;
    
    Ans = iNo1 - iNo2;

    return Ans
  }
}
    
class Program1
{
  public static void main(String A[])
  {
    int iValue1 = 30;
    int iValue2 = 10;
    int iRet = 0;
    
    Subtraction sobj = new Subtraction;

    iRet = sobj.Display(iValue1, iValue2);

    System.out.println("Subtraction is : ",+iRet);
  }
}
